pkg/apis/serving/v1alpha1/kfservice_validation_test.go

```diff
@@ -175,7 +175,7 @@ func TestCustomBadFields(t *testing.T) {
 			StdinOnce: true,
 		},
 	}
-	g.Expect(kfsvc.ValidateCreate()).Should(gomega.MatchError("Custom container validation error: must not set the field(s): name, stdin, stdinOnce"))
```

Comment:
Why did you remove name here?

Reply:
we are using Knative `ValidateContainer` which has `name` in the allowed list  https://github.com/knative/serving/blob/release-0.7/pkg/apis/serving/fieldmask.go#L194 (and was disallowed earlier)